events/wc_void_events.txt

```diff
@@ -380,8 +380,11 @@ character_event = {
 		is_dying = no
 		
 		NOT = { has_character_flag = void_mutation_ongoing_flag } # Prevents multiple firing
-
-		NOT = { trait = being_void }
+		
+		OR = {
+			NOT = { trait = being_void }
+			is_scale_race_trigger = yes
+		}
```

Comment:
What is this supposed to do?

Reply:
it checks for naga trait but i should do it in less roundabout way